packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -201,23 +212,27 @@ def _create_clients(
 
     has_multiple_service_classes = len(measurement_service_classes) > 1
     for service_class in measurement_service_classes:
-        base_service_class = extract_base_service_class(service_class)
-        if has_multiple_service_classes or module_name is None:
-            module_name = create_module_name(base_service_class, generated_modules)
-        if has_multiple_service_classes or class_name is None:
-            class_name = create_class_name(base_service_class)
-        validate_identifier(module_name, "module")
-        validate_identifier(class_name, "class")
-
-        _create_client(
-            channel_pool=channel_pool,
-            discovery_client=discovery_client,
-            measurement_service_class=service_class,
-            module_name=module_name,
-            class_name=class_name,
-            directory_out=directory_out_path,
-        )
-        generated_modules.append(module_name)
+        try:
+            base_service_class = extract_base_service_class(service_class)
+            if has_multiple_service_classes or module_name is None:
+                module_name = create_module_name(base_service_class, generated_modules)
+            if has_multiple_service_classes or class_name is None:
+                class_name = create_class_name(base_service_class)
+            validate_identifier(module_name, "module")
+            validate_identifier(class_name, "class")
+
+            _create_client(
+                channel_pool=channel_pool,
+                discovery_client=discovery_client,
+                measurement_service_class=service_class,
+                module_name=module_name,
+                class_name=class_name,
+                directory_out=directory_out_path,
+            )
+            generated_modules.append(module_name)
+        except Exception as e:
+            click.echo(CLIENT_CREATION_ERROR_MESSAGE.format(service_class, e))
+            continue
```

Comment:
nit: The `continue` statement skips the execution of the code block after the continue statement and forces the control to start the next iteration. So, having it at the end of the iteration has no effect.